app/models/concerns/dynamic_attribute.rb

```diff
@@ -7,6 +7,28 @@
 module DynamicAttribute
     extend ActiveSupport::Concern
     include Rails.application.routes.url_helpers
+    include ActionView::Helpers::DateHelper
+    include ActionView::Helpers::JavaScriptHelper
+    include ActionView::Helpers::RenderingHelper
+    include ActionView::Helpers::TagHelper
+    include ActionView::Helpers::UrlHelper
+    include ActionView::Helpers::TranslationHelper
+    include ActionView::Helpers::Tags
+    include ActionView::Helpers::NumberHelper
+    include ActionView::Helpers::AssetUrlHelper
+    include ActionView::Helpers::AssetTagHelper
+    include ActionView::Helpers::TextHelper
+    include ActionView::Helpers::DebugHelper
+    include ActionView::Helpers::CspHelper
+    include ActionView::Helpers::CsrfHelper
+    include ActionView::Helpers::AtomFeedHelper
+    include ActionView::Helpers::SanitizeHelper
+
+
```

Comment:
remove the following helpers: 

``` ruby
ActionView::Helpers::JavaScriptHelper
ActionView::Helpers::RenderingHelper
ActionView::Helpers::TagHelper
ActionView::Helpers::UrlHelper
ActionView::Helpers::Tags
ActionView::Helpers::AssetUrlHelper
ActionView::Helpers::AssetTagHelper
ActionView::Helpers::DebugHelper
ActionView::Helpers::CspHelper
ActionView::Helpers::AtomFeedHelper
ActionView::Helpers::SanitizeHelper
```

add tests that exercise 1 method each from: 

``` ruby
ActionView::Helpers::NumberHelper
ActionView::Helpers::TextHelper

```

make sure you add https://api.rubyonrails.org/classes/Time.html as well since we need to make time comparisons